### PR TITLE
[4.13] OCPBUGS-8710: Move PSa back to logging

### PR DESCRIPTION
**Pull request by stlaz**
Bumps the o/API with updated featuresets

/cc @deads2k @ibihim

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -23,6 +23,8 @@ require (
 	k8s.io/klog/v2 v2.80.1
 )
 
+replace github.com/openshift/api => github.com/stlaz/api v0.0.0-20230316150409-7dfb8e061a12
```
Comment (ibihim) [inline go.mod:26]: Is this intended to stay within the PR?

Comment (stlaz) [reply go.mod:26]: it's a fake bump, we can't merge until the API PR merges (which it did - I need to update the PR)

**Comment by stlaz** [comment]
/retest-required
I wonder if there are more things still broken in the build payloads

**Pull request merged**